uk.co.compucorp.civicrm.hrleaveandabsences/js/test/shared/controllers/request.controller.spec.js

```diff
@@ -645,12 +646,19 @@
           });
         });
 
-        describe('basic tests for when manager opens leave request popup in edit mode', function () {
+        describe('when manager opens someone else\'s leave request', function () {
```

Comment:
Using escape character here makes it less readable, instead please use double quotes outside
`"when manager opens someone else's leave request"`